docs/source/interfaces/direct/dns_powder_tof/dns_powder_tof_options_tab.rst

```diff
@@ -0,0 +1,33 @@
+.. _dns_powder_tof_options_tab-ref:
+
+DNS Powder TOF Options Tab
+--------------------------
+
+.. image::  ../../../images/DNS_interface_powder_tof_options_tab.png
+   :align: center
+   :height: 400px
+\
+
+This tab of the powder TOF mode defines the options for the reduction script.
+The most useful configuration is preselected so that one just needs to set the
+binning and proceed to the next tab if no change to the selected options is
+desired.
+
+The **Wavelength** value can be overwritten, for example for the cases when
+:math:`\lambda/2` was selected by the velocity selector.
+
+**Detector Efficiency Correction** norms the detector efficiency using standard
+vanadium data.
+
+Instrument background can be subtracted from vanadium and sample data,
```

Comment:
Each entry here should begin with the name of the option in the gui, e.g. `**Subtract Instrument Background from Vanadium** can be enabled to subtract the instrument background from vanadium and sample data.`

Also does the subtraction happen automatically from the sample data or does it have to be enabled through this option, if so the name of the option should be changed to show that it also subtracts from the sample.

Reply:
> Also does the subtraction happen automatically from the sample data or does it have to be enabled through this option, if so the name of the option should be changed to show that it also subtracts from the sample.

The documentation for this option is misleading. This check mark can be used to enable background subtraction from vanadium data only. The documentation will be updated correspondingly